mava/wrappers/jumanji.py

```diff
@@ -123,68 +129,85 @@ def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
 class ConnectorWrapper(MultiAgentWrapper):
     """Multi-agent wrapper for the MA Connector environment."""
 
-    def __init__(self, env: MaConnector):
+    def __init__(self, env: MaConnector, has_global_state: bool = True):
         super().__init__(env)
+        self.has_global_state = has_global_state
 
-    def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
+    def modify_timestep(
+        self, timestep: TimeStep
+    ) -> TimeStep[Union[Observation, ObservationGlobalState]]:
         """Modify the timestep for the Connector environment."""
 
         def create_agents_view(grid: chex.Array) -> chex.Array:
-            positions = jnp.where(grid % 3 == 2, True, False)
-            targets = jnp.where((grid % 3 == 0) & (grid != 0), True, False)
-            paths = jnp.where(grid % 3 == 1, True, False)
-            my_position = jnp.where(grid == 2, True, False)
-            my_target = jnp.where(grid == 3, True, False)
+            positions = jnp.where(grid % TARGET == POSITION, True, False)
+            targets = jnp.where((grid % TARGET == 0) & (grid != EMPTY), True, False)
+            paths = jnp.where(grid % TARGET == PATH, True, False)
```

Comment:
Sorry for the confusion, but `grid % TARGET` is a bit confusing to me and I see there isn't a better constant in connector for this, but technically `TARGET` isn't gaurenteed to always be the largest item e.g if we added another item to connector in the future, so I would put `n_grid_items = 3` at the top of `modify_timestep` and always ` % n_grid_items` that should make it super clear what you're doing :+1:

Reply:
Its a bit tricky since in that case TARGET wouldn't appear at all. Ive added a comment that clarifies things.